third_party/maya/lib/usdMaya/MayaMeshWriter.cpp

```diff
@@ -38,6 +38,49 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 PXRUSDMAYA_REGISTER_ADAPTOR_SCHEMA(MFn::kMesh, UsdGeomMesh);
 
+namespace {
+
+void _exportReferenceMesh(UsdGeomMesh& primSchema, MObject obj) {
+    MStatus status = MS::kSuccess;
+    MFnDependencyNode dNode(obj, &status);
+    if (!status) { return; }
+
+    auto referencePlug = dNode.findPlug("referenceObject", &status);
+    if (!status || referencePlug.isNull()) { return; }
+
+    MPlugArray conns;
+    referencePlug.connectedTo(conns, true, false);
+    if (conns.length() == 0) { return; }
+
+    auto referenceObject = conns[0].node();
+    if (!referenceObject.hasFn(MFn::kMesh)) { return; }
+
+    MFnMesh referenceMesh(referenceObject, &status);
+    if (!status) { return; }
+
+    const auto* mayaRawPoints = referenceMesh.getRawPoints(&status);
+    const auto numVertices = referenceMesh.numVertices();
+    VtArray<GfVec3f> points(numVertices);
+    for (auto i = decltype(numVertices){0}; i < numVertices; ++i) {
+        const auto floatIndex = i * 3;
+        points[i].Set(mayaRawPoints[floatIndex],
+                        mayaRawPoints[floatIndex + 1],
+                        mayaRawPoints[floatIndex + 2]);
+    }
+
+    static const TfToken _prefToken("Pref");
```

Comment:
We think it'd be good to expose this attribute name outside of usdMaya so that consumers throughout the pipeline can be consistent. One idea is to add a helper function in usdUtils/pipeline.h similar to:

https://github.com/PixarAnimationStudios/USD/blob/56c2e1def36ee07121f4ecb349c1626472b3c338/pxr/usd/lib/usdUtils/pipeline.h#L109-L112

Maybe `UsdUtilsGetPrefName()`?